python/ambassador_diag/diagd.py

```diff
@@ -481,11 +486,24 @@ def show_overview(reqid=None):
                  banner_content=banner_content,
                  **ov, **ddict)
 
+    patch_client = request.args.get('patch_client', None)
     if request.args.get('json', None):
         key = request.args.get('filter', None)
 
         if key:
             return jsonify(tvars.get(key, None))
+        elif patch_client:
+            # Get the previous full representation
+            cached_tvars_json = tvars_cache.get(patch_client, "{}")
```

Comment:
Is there a reason to save the JSON serialization of the dict, rather than the dict itself? It seems that saving the dict itself would save you a `json.loads()` later...

Reply:
True, let me test that real quick.